.github/workflows/kcp-image.yaml

```diff
@@ -60,14 +54,14 @@ jobs:
             type=raw,value=${{ env.sha_short }}
```

Comment:
Not in the direct scope, but maybe an opportunity to have that replaced with `type=sha,prefix=,suffix=,format=short` so the logic that computes / sets `sha_short` can be removed.

Reply:
done ptal